upstairs/src/downstairs.rs

```diff
@@ -2530,22 +2529,19 @@ impl Downstairs {
             }
         }
 
-        // Make sure this job is counted for backpressure
-        let is_write = matches!(io.work, IOop::Write { .. });
-
-        // Puts the IO onto the downstairs work queue.
+        let is_write =
+            matches!(io.work, IOop::Write { .. } | IOop::WriteUnwritten { .. });
```

Comment:
Should places that match against either IOop instead call `work.is_write` or something equivalent?

Reply:
Added `IOop::is_write` in 28f79c258115, and used it in a few places.